Homework_Concurrency/src/main/java/MainFestival.java

```diff
@@ -0,0 +1,14 @@
+public class MainFestival {
+    public static FestivalStatisticThread festivalStats;
+
+    public static void main(String[] args) {
+        Gate gate = new Gate();
+        festivalStats = new FestivalStatisticThread(gate);
+        for (int i = 0; i <500; i++)
```

Comment:
not unlimited, should be with random of a huge number or while(true) flow / infinite loop